phys2bids/utils.py

```diff
@@ -148,36 +148,6 @@ def check_file_exists(filename):
     if not os.path.isfile(filename) and filename is not None:
         raise FileNotFoundError(f'The file {filename} does not exist!')
 
```

Comment:
Can you add a newline after this one? The style we're adopting asks for two lines between every function.